hack/catalog.sh

```diff
@@ -24,8 +24,8 @@ find $OLM_DIR -name '*_crd.yaml' | sort -n | xargs -I{} cp {} $CRD_DIR/
 
 # Determine if we're running locally or in CI.
 if [ -n "$OPENSHIFT_BUILD_NAMESPACE" ]; then
```

Comment:
we should probably not even depend on existence of OPENSHIFT_BUILD_NAMESPACE then? (not mentioned here https://github.com/openshift/ci-tools/blob/master/TEMPLATES.md#image_format )

Reply:
Let's do that separately. A lot of the variables actually mentioned in this doc are not available to our jobs so I asked for guidance here.

Reply:
ok